packages/orbit-components/src/common/placements.ts

```diff
@@ -0,0 +1,39 @@
+import type { Placement as FloatingPlacement, Alignment } from "@floating-ui/react";
+
+export type AutoPlacement = "auto" | "auto-start" | "auto-end";
+export type Placement = FloatingPlacement;
```

Comment:
Maybe aliasing here would remove some amount of code changes elsewhere
```suggestion
export type FixedPlacement = FloatingPlacement;
export type Placement = FixedPlacement | AutoPlacement;
```

Reply:
Yes, I did this one. But the enums I couldn't, because I don't think we can compose enums from other enums 🤔